credoai/evaluators/shap.py

```diff
@@ -104,16 +105,16 @@ def _validate_arguments(self):
     def evaluate(self):
         ## Overall stats
         self._setup_shap()
-        res = self._get_overall_shap_contributions()
-        self.results = [TableContainer(res, **self.get_container_info())]
+        self.results = [
+            TableContainer(
+                self._get_overall_shap_contributions(), **self.get_container_info()
+            )
+        ]
 
         ## Sample specific results
         if self.samples_ind:
-            labels = {"ordered_feature_names": self.shap_values.feature_names}
             ind_res = self._get_mult_sample_shapley_values()
-            self.results += [
-                TableContainer(ind_res, **self.get_container_info(labels=labels))
-            ]
+            self.results += [TableContainer(ind_res, **self.get_container_info())]
```

Comment:
To confirm, passing feature names to results container no longer needed because everything is handled in DataFrames directly below?